lib/analytics.js

```diff
@@ -114,13 +126,18 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(settings, o
     if (!Integration) delete settings[name];
   }, settings);
 
-  // add integrations
-  each(function(opts, name) {
-    var Integration = self.Integrations[name];
-    var integration = new Integration(clone(opts));
-    self.log('initialize %o - %o', name, opts);
-    self.add(integration);
-  }, settings);
+  if (doNotTrack) {
+    // Clear manually added integrations
+    this._integrations = {};
```

Comment:
ah i definitely saw this as only sending `context.doNotTrack` when doNotTrack is on, which would defeat the purpose. we are adding `context.doNotTrack` as a field to all messages, regardless of the flag. got it 👌